index.bs

```diff
@@ -2771,24 +2797,64 @@ To <dfn>parse aggregatable trigger data</dfn> given a [=map=] |map|:
     1. [=list/Append=] |aggregatableTrigger| to |aggregatableTriggerData|.
 1. Return |aggregatableTriggerData|.
 
-To <dfn>parse aggregatable key-values</dfn> given a [=map=] |map|:
+To <dfn>parse aggregatable filtering id max bytes</dfn> given a [=map=] |map|:
 
+1. Let |maxBytes| be [=default filtering ID max bytes=].
+1. If |map|["`aggregatable_filtering_id_max_bytes`"] [=map/exists=]:
+    1. Let |maxBytesValue| be |map|["`aggregatable_filtering_id_max_bytes`"].
+    1. If |maxBytesValue| is a non-negative integer and is [=set/contained=] in the [=valid filtering ID max bytes range=].
+        Let |maxBytes| be |maxBytesValue|.
+    1. Otherwise, return null.
+1. Return |maxBytes|.
+
+To <dfn>parse aggregatable key-values value</dfn> given a |value|:
+1. If |value| is not an integer, return null.
+1. If |value| is less than or equal to 0, return null.
+1. If |value| is greater than [=allowed aggregatable budget per source=], return null.
+1. Return |value|.
+
+To <dfn>parse aggregatable key-values</dfn> given a [=map=] |map| and integer |maxBytes|:
```

Comment:
Should we require `maxBytes` to be positive?

Reply:
might as well, updated to `a positive integer |maxBytes|`.